test/kubernetes/testutils/actions/provider/provider.go

```diff
@@ -0,0 +1,54 @@
+package provider
+
+import (
+	"testing"
+
+	"github.com/solo-io/gloo/test/kubernetes/testutils/actions/glooctl"
+	"github.com/solo-io/gloo/test/kubernetes/testutils/actions/kubectl"
+
+	"github.com/solo-io/gloo/test/kubernetes/testutils/cluster"
+	"github.com/solo-io/gloo/test/kubernetes/testutils/gloogateway"
+)
+
+// ActionsProvider is the entity that creates actions.
+// These actions are executed against a running installation of Gloo Gateway, within a Kubernetes Cluster.
+// This provider is just a wrapper around sub-providers, so it exposes methods to access those providers
+type ActionsProvider struct {
+	kubeCtlProvider *kubectl.Provider
+	glooCtlProvider glooctl.Provider
```

Comment:
nit: these are not camel case anywhere else that I have ever seen
```suggestion
	kubectlProvider *kubectl.Provider
	glooctlProvider glooctl.Provider
```